libuser/src/ipc/mod.rs

```diff
@@ -179,18 +179,53 @@ pub struct IPCBuffer<'a> {
 pub trait SizedIPCBuffer {
     /// Return the size of the type.
     fn size(&self) -> usize;
+
+    /// Check if the address and size are correct.
+    fn is_cool(addr: usize, size: usize) -> bool;
+
+    /// Create a reference to a ipc buffer from part.
```

Comment:
```suggestion
    /// Create a reference to a ipc buffer from an address and a byte size.
```

Also might be worth linking to https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html